markdown/ch1.markdown

```diff
@@ -46,9 +46,9 @@ def rpkm(counts, lengths):
     normed : array, shape (N_genes, N_samples)
         The RPKM normalized counts matrix.
     """
-    N = np.sum(counts, axis=0)  # sum each column to get total reads per sample
+    C = counts.astype(float)  # use float to avoid overflow with `1e9 * C`
```

Comment:
This forward reference is a bit confusing.  Can you add a slightly clearer comment?